### DISCUSSION TITLE
希望能增加针对树莓派的安装包

### DISCUSSION THREAD

**Discussion by x0chaoren0s** [BeyondDimension/SteamTools#1200] [Ideas]
希望能增加针对树莓派架构的安装包，树莓派架构是armhf

`
pi@raspberrypi:~/Downloads $ sudo dpkg -i 'Steam  _linux_arm64_v2.6.9.deb'
`
`
dpkg: 处理归档 Steam  _linux_arm64_v2.6.9.deb (--install)时出错：
`
`
 软件包体系结构(arm64)与本机系统体系结构(armhf)不符
`
`
在处理时有错误发生：
`
`
 Steam  _linux_arm64_v2.6.9.deb
`

**Comment by AigioL** [comment]
因为在 树莓派 ARM32 位的设备上运行体验不好，可能受限于硬件 GPU 性能导致的卡顿，当前的版本中没有编译 ``linux-arm`` 的版本
```linux-arm64``` 需要在 64 位 ARM 上运行的 Linux 发行版本，如 Raspberry Pi Model 3 及更高版本上的 Ubuntu 服务器 64 位

Raspberry Pi 3 Model B 使用的是 Broadcom BCM2837 SoC 看上去是支持 64 位的，你应该装一个 ARM64 位的系统
例如 Ubuntu 20.04 LTS arm64
https://ubuntu.com/certified/201811-26649#:~:text=Ubuntu%20Core%2020%20arm64%20%20%20Network%20,drm-platform-soc_gpu%20%20%20Wireless%20%20%20mmcnr%20

**Comment by Mossimos** [comment]
树莓派 不推荐使用 4以下的设备了  我今天刚刚试了下 deb 包都需要解压个5分钟

**Comment by Mossimos** [comment]
![4107TNPU_9A~ZKLM }CJ70B](https://user-images.githubusercontent.com/26396340/161935935-0d98b51f-874e-4e78-91f6-71e346ca6b80.png)
![QQ截图20220406165144](https://user-images.githubusercontent.com/26396340/161935964-00eb2713-e0f7-4e93-9aa4-5d3b2a2a20e6.png)
如果你能接受点一下按钮 大约 10s 程序才有反应可以在 低于 树莓派4 一下的设备使用 gui 版本
